test/controller/add_admin_test.go

```diff
@@ -47,151 +47,71 @@ var _ = Describe("addAdmin", func() {
 
 	When("controller owner calls AddAdmin with it's own address", func() {
 
-		var tx *types.Transaction
-		const gasLimit = 100000
-
-		BeforeEach(func() {
-			var err error
-			tx, err = ControllerContract.AddAdmin(ControllerOwner.TransactOpts(ethertest.WithGasLimit(gasLimit)), ControllerOwner.Address())
+		It("should fail at the already owner requirenment", func() {
+			tx, err := ControllerContract.AddAdmin(ControllerOwner.TransactOpts(ethertest.WithGasLimit(100000)), ControllerOwner.Address())
 			Expect(err).ToNot(HaveOccurred())
 			Backend.Commit()
-		})
-
-		It("should not succeed", func() {
 			Expect(isSuccessful(tx)).To(BeFalse())
-		})
-
-		It("should not exaust gas", func() {
-			Expect(isGasExhausted(tx, gasLimit)).To(BeFalse())
-		})
-
-		It("should fail at the already owner requirenment", func() {
-			Expect(TestRig.LastExecuted()).To(MatchRegexp(`require\(!_isOwner\(_account\), "provided account is already the owner"\);`))
+			returnData, _ := ethCall(tx)
+			Expect(string(returnData[len(returnData)-64:])).To(ContainSubstring("provided account is already the owner"))
 		})
 	})
 
 	When("controller owner calls AddAdmin with controller's address", func() {
 
-		var tx *types.Transaction
-		const gasLimit = 100000
-
-		BeforeEach(func() {
-			var err error
-			tx, err = ControllerContract.AddAdmin(ControllerOwner.TransactOpts(ethertest.WithGasLimit(gasLimit)), Controller.Address())
+		It("should fail at already controller requirenment", func() {
+			tx, err := ControllerContract.AddAdmin(ControllerOwner.TransactOpts(ethertest.WithGasLimit(100000)), Controller.Address())
 			Expect(err).ToNot(HaveOccurred())
 			Backend.Commit()
-		})
-
-		It("should not succeed", func() {
 			Expect(isSuccessful(tx)).To(BeFalse())
-		})
-
-		It("should not exaust gas", func() {
-			Expect(isGasExhausted(tx, gasLimit)).To(BeFalse())
-		})
-
-		It("should fail at already controller requirenment", func() {
-			Expect(TestRig.LastExecuted()).To(MatchRegexp(`require\(!_isController\[_account\], "provided account is already a controller"\);`))
+			returnData, _ := ethCall(tx)
+			Expect(string(returnData[len(returnData)-64:])).To(ContainSubstring("provided account is already a controller"))
```

Comment:
These expected strings have the a/an articles - why remove them from only two require statements?